### ISSUE TITLE
"Mark as watched" on episode is not working like it should

### ISSUE THREAD

**Issue by soredake** [michaldrabik/showly#720]
**Describe the bug**
When i mark episode as watched the show does not show up in the progress and episode watch is not synced back to trakt.tv.

https://github.com/michaldrabik/showly-2.0/assets/5204968/ba2f29bf-8d1a-40f8-b975-8702da8d05a6

**Expected behavior**
TV show should show up in progress after marking episode as watched, as in official app.

https://github.com/michaldrabik/showly-2.0/assets/5204968/ea88f89f-7799-496b-a53c-0e0e1d275701

**Comment by michaldrabik** [comment]
Make sure the show is added into My Shows. Only then will it be included in progress.

**Comment by soredake** [comment]
I see, it's how showly works in comparison with official app.

**Comment by michaldrabik** [comment]
Yes because Trakt does not recognize such an option where you can have a show followed but 0 episodes watched.

**Comment by soredake** [comment]
>Yes because Trakt does not recognize such an option where you can have a show followed but 0 episodes watched.

But after pressing "mark as watched" on episode one for example it should have 1 episode watched.
